.circleci/config.yml

```diff
@@ -37,19 +37,26 @@ jobs:
       - run: make prove-docker
   build:
     docker:
-      - image: quay.io/3scale/s2i-openresty-centos7:1.11.2.5-1
+      - image: quay.io/3scale/s2i-openresty-centos7:junit
```

Comment:
This needs to be changed to new released version.

Reply:
Will do a release after https://github.com/3scale/s2i-openresty/pull/37 gets merged.